antq.sh

```diff
@@ -26,6 +26,33 @@ version_ge() {
 version_gt() { 
     test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" != "$1"; 
 }
+version_lt() { 
+    test "$(echo "$@" | tr " " "\n" | sort -V | head -n 1)" == "$1"; 
+}
+
+close_outdated_pr() {
+    existingPrs=$(gh pr list --state open --json title,number | jq -c '.[]')
```

Comment:
```suggestion
    existingPrs=$(gh pr list --state open --limit 1000 --json title,number | jq -c '.[]')
```
The default seems to be 30, so some PRs could be missed.